### PR TITLE
Add single click mode

### PR DESCRIPTION
**Pull request by Namaneo**
This PR adds the ability to navigate through the files only using single clicks, using checkboxes to select one or multiple elements. As it directly relates to the user experience, this is configurable per user instead of globally. In my opinion, this improves the user experience on mobile, and that's the main reason I propose those changes. 🙂 

Though, some questions might need to be addressed:
* Would it be a better idea to enable it on mobile and disable it on desktop, instead of manually configuring it?
* I used raw checkboxes, do you want to replace them with Material looking ones instead? I didn't want to alter the package dependencies before asking.

### CODE REVIEW

**Comment by ramiresviana** [comment]
The single click idea is good, is the common behavior on mobile apps, but the checkbox dont fit on the listing, is too small for mobile, and is similar to an existing feature, the multiple selection mode that have a button in the header bar. I think we should improve the multiple selection mode to work better on mobile, having a better interface and toggling.

**Comment by Namaneo** [comment]
Oh my bad, I'm pretty new to this project and haven't even seen the multi-selection button. Then, we could just remove the checkboxes to only rely on this button. The configuration entry would concern the single-click behavior on mobiles only. Desktop behavior would stay the same as before. What do you think?

**Comment by ramiresviana** [comment]
Filebrowser on mobile and desktop is the same application and behaves in the same way for both those variations. Adding a feature that changes the behavior only on mobile can be bad for the project maintainability and can make users confused. Just let the users choose if they want to enable the feature.

**Comment by Namaneo** [comment]
Alright, then I will rename the configuration option to something like "Use single clicks to open folders" instead, this will be more accurate. I will also remove the checkboxes: in this mode, only the top-right button will trigger the multi-selection.

File: cmd/users.go [hunk 1]
```diff
@@ -30,12 +30,13 @@ func printUsers(usrs []*users.User) {
 	fmt.Fprintln(w, "ID\tUsername\tScope\tLocale\tV. Mode\tAdmin\tExecute\tCreate\tRename\tModify\tDelete\tShare\tDownload\tPwd Lock")
```
Comment (ramiresviana) [inline cmd/users.go]:
```suggestion
	fmt.Fprintln(w, "ID\tUsername\tScope\tLocale\tV. Mode\tS.Click\tAdmin\tExecute\tCreate\tRename\tModify\tDelete\tShare\tDownload\tPwd Lock")
```

File: frontend/src/i18n/en.json [hunk 2]
```diff
@@ -173,6 +173,7 @@
     "globalRules": "This is a global set of allow and disallow rules. They apply to every user. You can define specific rules on each user's settings to override this ones.",
     "allowSignup": "Allow users to signup",
     "createUserDir": "Auto create user home dir while adding new user",
+    "singleClick": "Use single clicks to open folders",
```
Comment (ramiresviana) [inline frontend/src/i18n/en.json]:
```suggestion
    "singleClick": "Use single clicks to open files and directories",
```

**Review by ramiresviana — CHANGES_REQUESTED** [review]

File: frontend/src/i18n/en.json (hunk 2 above)
Comment (Namaneo) [reply frontend/src/i18n/en.json]: I will check in the evening, but I think those changes does not affect files, only directories.
I can rename `folders` to `directories` anyway. 🙂 

EDIT: Aaaand I'm wrong, it applies to files as well!

**Comment by o1egl** [comment]
@Namaneo can you please resolve merge conflict?

**Review by ramiresviana — APPROVED** [review]

**Pull request merged**